cylc/rose/utilities.py

```diff
@@ -87,6 +87,13 @@ def get_rose_vars_from_config_node(config, config_node, environ):
     for section in ['env', templating]:
         # Add standard ROSE_VARIABLES
         config_node[section].set(['ROSE_SITE'], rose_site)
+        if 'ROSE_VERSION' in config_node[section].value:
+            user_rose_version = config_node[section].value['ROSE_VERSION']
```

Comment:
Do we need to remove this variable from the config in order to prevent it from being exported?

Reply:
We replace it in the config with `metomi.rose.__version__` which _is_ exported. Do we want this or not?